web/src/index.css

```diff
@@ -24,8 +25,9 @@
   --bright-red: #EB5757;
   --red: #DB4437;
   --pale-gray: #EFEFEF;
-  --light-gray: #BDBDBD;
-  --gray: #A0A0A0;
+  --pale-gray: #BDBDBD;
```

Comment:
why are there 2 different `pale-gray`s

Reply:
omg thanks for noticing LOL I copy pasted and forgot to replace

Reply:
HAHA okay!